qiskit_experiments/library/calibration/__init__.py

```diff
@@ -62,10 +66,11 @@
 """
 
 from .drag import DragCal
+from .fine_drag import FineDrag, FineXDrag, FineSXDrag
 from .rabi import Rabi, EFRabi
 from .fine_amplitude import FineAmplitude, FineXAmplitude, FineSXAmplitude
 from .ramsey_xy import RamseyXY
 
 from .analysis.drag_analysis import DragCalAnalysis
-from .analysis.fine_amplitude_analysis import FineAmplitudeAnalysis
```

Comment:
I don't think so since `ErrorAmplificationAnaysis` moved to `curve_analysis.standard_analysis`. What would be the advantage of importing it here again?

Reply:
It seems like all the analysis classes for calibration experiments are being imported into this namespace here. The fine amplitude analysis class used to be imported here but now `ErrorAmplificationAnalysis` is the fine amplitude analysis class and it's not imported. The way it is now is fine if you are happy with it.

Another option might be a `FineAmplitudeAnalysis` class that subclasses `ErrorAmplificationAnalysis` without modification. That is a little weird but it might match user expectations of the naming conventions for experiments and analysis classes and allow `FineAmplitudeAnalysis` to be decoupled from the parent class if that were ever useful.

Reply:
Here is an approach that I like: 80b4625 . Remove the fixed parameters from `ErrorAmplificationAnalysis` then define these in the subclasses like `FineAmplitudeAnalysis`. This a) leaves `ErrorAmplificationAnalysis` more general and b) is more symmetric with the `FineDragAnalysis`.